testing/test_data_manager.py

```diff
@@ -0,0 +1,188 @@
+            kpi_json_ref = json.loads(f.read()) 
+        self.assertDictEqual(self.case.kpi_json, kpi_json_ref)
+        
+        # Check the content of the data loaded
+        df_man = self.case.data
+        self.compare_ref_timeseries_df(df_man, self.ref_data_loaded)
+
+    def test_get_data_default(self):
+        '''Check that the data manager can retrieve the test case data
+        for the default horizon and interval.
+         
+        '''       
+        
+        # Get the data
+        data_dict = self.man.get_data()
+        
+        # Check the data retrieved with the manager
+        df_man = pd.DataFrame(data_dict).set_index('time')
+        self.compare_ref_timeseries_df(df_man, self.ref_data_default)
+     
+    def test_get_data_index(self):
+        '''Check that the data manager can retrieve the test case data
+        when an arbitrary time index is provided.
+         
+        '''       
+        
+        # Define index
+        index = np.arange(0, 7*24*3600, 321)
+         
+        # Get the data
+        data_dict = self.man.get_data(index=index)
+        
+        # Check the data retrieved with the manager
+        df_man = pd.DataFrame(data_dict).set_index('time')
+        self.compare_ref_timeseries_df(df_man, self.ref_data_index)
+    
+class DataManagerSingleZoneTest(unittest.TestCase, utilities.partialChecks,
+                      PartialDataManagerTest):
+    '''Tests the data manager class in a single-zone example. 
+     
+    '''
+ 
+    def setUp(self):
+        '''Setup for each test.
+         
+        '''
+        
+        # Mimic testcase2 as in boptest container
+        utilities.create_mimic_boptest('testcase2')
+        os.chdir(utilities.get_root_path())
+        self.case=testcase.TestCase()
+        
+        # Instantiate a data manager
+        self.man = Data_Manager(self.case)
+        
+        # Set reference file paths
+        self.ref_kpis = os.path.join(testing_root_dir, 
+            'references', 'data', 'testcase2', 'kpis.json')
+        self.ref_data_loaded = os.path.join(testing_root_dir, 
+            'references', 'data', 'testcase2', 'tc2_data_loaded.csv')
+        self.ref_data_default = os.path.join(testing_root_dir,
+            'references', 'data', 'testcase2', 'tc2_data_retrieved_default.csv')
+        self.ref_data_index = os.path.join(testing_root_dir,
+            'references', 'data', 'testcase2', 'tc2_data_retrieved_index.csv')        
+
+    def tearDown(self):
+        '''Tear down for each test.
+        
+        '''
+        
+        utilities.remove_mimic_boptest()
+
+class DataManagerMultiZoneTest(unittest.TestCase, utilities.partialChecks,
+                               PartialDataManagerTest):
+    '''Tests the data manager class in a single-zone example.  
+     
+    '''
+ 
+    def setUp(self):
+        '''Setup for each test.
+         
+        '''
+        
+        # Mimic testcase2 as in boptest container
```

Comment:
`# Mimic testcase3 as in boptest container`

Reply:
Fixed in https://github.com/ibpsa/project1-boptest/pull/229/commits/e43274c8a7e64eadf9de962f64d74d57ce1e57a8.